icons/fabric.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M22 6H12V2H6v10H2v6h10v4h6V12h4Z" />
```

Comment:
Could be misconstrued as a hate symbol.

Reply:
?

Reply:
?

Reply:
I think @jguddas means a swastika. I think we're in the safe, but as usual: we need use cases.

Currently, I don't really see why we have to have `fabric` _and_ `cloth`, why not pick just one?
How exactly would the use cases differ?

Reply:
> I think @jguddas means a swastika.

I see. I guess we can add _completely ruined any future kind of quad-based pattern_ on top of their other horrific crimes… 😅

Yeah I think it’s fine if the existing https://lucide.dev/icon/fan is ok…

> don't really see why we have to have `fabric` and `cloth`

My thinking was that while `cloth` is more specific to the others here, `fabric` can also be used as a more generic way to represent ‘pattern’ and also ‘holding things together’ in general… For example, I have an npm package called `fabric` (currently private) that bundles together all of my sharable configs for the multitude of tools frequently used across similar projects…

Reply:
That sounds suspiciously like a brand icon request to me. 😱

Can we maybe try something more along the line of "pattern"?

Reply:
> That sounds suspiciously like a brand icon request to me.

Definitely not a brand 😅 Just an example of a ‘holding things together’ use case…

> more along the line of "pattern"?

Yeah the primarily use case is already ‘pattern’, which I think it does convey quite well… Particularly since it tessellates nicely:

<img width="176" alt="Untitled" src="https://github.com/lucide-icons/lucide/assets/7797479/c310ed5a-6724-41ba-890b-c0f72008d7e6">

I was trying to cover both use cases in a single, more generic icon…

Reply:
> I see. I guess we can add completely ruined any future kind of quad-based pattern

Yes, don't do it.

![image](https://github.com/lucide-icons/lucide/assets/25524993/80a18ed3-d38a-4c7e-95eb-def0f0203b41) 🤮

Reply:
> > I see. I guess we can add completely ruined any future kind of quad-based pattern
> 
> Yes, don't do it.
> 
> ![image](https://user-images.githubusercontent.com/25524993/238144088-80a18ed3-d38a-4c7e-95eb-def0f0203b41.png) 🤮

Didn’t even notice that in a hurry 😅 The bastards!

Still it is standalone…